fccodegen/tests/test_weblab_templating.py

```diff
@@ -36,7 +52,7 @@ def test_generate_weblab_model(tmp_path):
     generated = path.read_text()
 
     # Store locally to update test output file
-    if True:
+    if False:
```

Comment:
I now tend to make lines like this read an environment variable, so I can just do e.g. `WEBLAB_REGENERATE_REF=1 pytest` to regenerate all reference data.

Reply:
Done!